en/drafts/originals/agent-based-model-communication-networks.md

```diff
@@ -724,17 +726,15 @@ else:
     )[0]
 ```
 
-If agents have already received some letters, the likeliness of receiving more letters grows. In this way, we can allow agents to become, in a sense, more "famous". This could be one simple possible mechanism to model why well-known people like Christiaan Huygens seem to have received much more letters than others.
-
-We also make it less likely for every agent to move in every step, as people don't constantly relocate. For this, we introduce another weighted random choice, this time with fixed weight. Now agents will only move with a chance of 20%, whenever they draw a one.
+We'll also reduce the rate of agents moving to different cell at every step, as people don't constantly relocate. For this, we introduce another weighted random choice, this time with fixed weight. Now, the chance of an agent moving will only be 20%, or whenever they draw a '1'.
 
 ```python
 if self.random.choices([0,1], weights=[0.8, 0.2], k=1)[0] == 1:
     new_position = self.random.choice(possible_steps)
     self.model.grid.move_agent(self, new_position)
 ```
 
-To be able to initialize the agents with this new option we also have to add another parameter to the model itself. All together we get the following new definitions for agents and the model.
+To initialize the agents with this new option, we have to add a final parameter to the model itself. All together, we get the following new definitions for agents and model:
```

Comment:
You had written 'we also have to add another parameter' – what is this parameter?

Reply:
The parameter is reinforce = False to switch between two modes of running the model. I suggest to add at the end of the sentence:
 
"To initialize the agents with this new option, we have to add a final parameter to the model itself, which controls if the reinforcement process should be used for selecting letter receivers." 

Sorry I could not figure out how to suggest elegantly...